oxd-server/src/test/java/org/gluu/oxd/server/GetIssuerTest.java

```diff
@@ -0,0 +1,65 @@
+package org.gluu.oxd.server;
+
+import org.gluu.oxd.client.ClientInterface;
+import org.gluu.oxd.common.params.GetIssuerParams;
+import org.gluu.oxd.common.response.GetIssuerResponse;
+import org.gluu.oxd.server.Tester;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertNotNull;
+
+public class GetIssuerTest {
+    private static final String EMAIL_ADDRESS_INPUT = "admin@jenkins-ldap.gluu.org";
+    private static final String URL_INPUT = "https://jenkins-ldap.gluu.org/admin";
+    private static final String HOSTNAME_PORT_INPUT = "jenkins-ldap.gluu.org";
```

Comment:
It shouldn't be hardcoded inside test. When server will be changed, it will change too. Please parametrize via testng.xml. Same for generated client test.

Reply:
Done